.github/workflows/add-codeql-prs-to-project.yml

```
@@ -0,0 +1,22 @@
+name: Add CodeQL pull requests to project
+
+on:
+  pull_request:
+    types:
+      - opened
+      - labeled
+
+jobs:
+  add-codeql-static-prs-to-project:
+    name: Add CodeQL static team PRs to project
+    runs-on: ubuntu-latest
+    permissions:
```

Try this. The condition here should ensure the entire job only runs when the PR has a relevant label. This keeps the compute usage low and avoids cluttering the Actions tab for this repo.
See https://docs.github.com/en/actions/learn-github-actions/expressions#example-using-an-object-filter for docs on this syntax. This is a bit clunky, because there is no subset operator in the expression language and we have to list all the possible labels.
```suggestion
    runs-on: ubuntu-latest
    if: |
      contains(github.event.pull_request.labels.*.name, 'C#') ||
      contains(github.event.pull_request.labels.*.name, 'Go') ||
      contains(github.event.pull_request.labels.*.name, 'Java') ||
      contains(github.event.pull_request.labels.*.name, 'Kotlin')
    permissions:
```